model_lab_configs/huggingface/microsoft/Phi-3.5-mini-instruct/1/inference_sample.ipynb

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
we support copy now. if they are same, maybe consider it [windows-ai-studio-templates/model_lab_configs/huggingface/google/vit-base-patch16-224/1/_copy.json.config at hualxie/demo · microsoft/windows-ai-studio-templates](https://github.com/microsoft/windows-ai-studio-templates/blob/hualxie/demo/model_lab_configs/huggingface/google/vit-base-patch16-224/1/_copy.json.config)